scripts/setup/delete-reinitialize-database

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env sh
+
+set -e
+set -x
+
+cat <<EOF
+This script will completely wipe your database, including all users, chat
+messages, and metadata.  It will not modify your existing /etc/zulip
+configuration files, but all other database-backed data will disappear. This
+script is targeting /home/zulip/deployments/current.
+
+Do you want to continue? You must type YES.
+EOF
+
+read -r yesno
+
+if [ "x$yesno" != "xYES" ]; then
+  >&2 echo "Exiting without changes."
+  exit 1;
+fi
+
+if [ "$(id -u)" -ne "0" ]; then
+  >&2 echo "Exiting because root is not available."
```

Comment:
1. This check should happen first before everything else (see https://github.com/zulip/zulip/blob/master/scripts/setup/install).
2. The message should be standardized according to https://github.com/zulip/zulip/blob/cc54dddd84fc49996fe561efc371cae63e567871/scripts/lib/zulip_tools.py#L401, "{script_name} must be run as root.".